File: app/src/main/java/de/ironjan/mensaupb/app_info/About.java
package de.ironjan.mensaupb.app_info;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.v4.app.NavUtils;
import android.support.v7.app.AppCompatActivity;

import org.androidannotations.annotations.EActivity;
import org.androidannotations.annotations.OptionsItem;

import de.ironjan.mensaupb.R;

/**
 * Wrapping activity for {@link AboutFragment}.
 */
@SuppressLint("Registered")
@EActivity(R.layout.activity_about)
public class About extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }

    @OptionsItem(android.R.id.home)
    void navUp() {
        NavUtils.navigateUpFromSameTask(this);
    }
}

File: app/src/main/java/de/ironjan/mensaupb/app_info/About.kt
package de.ironjan.mensaupb.app_info

import android.annotation.SuppressLint
import android.os.Bundle
import android.support.v4.app.NavUtils
import android.support.v7.app.AppCompatActivity
import de.ironjan.mensaupb.R
import org.androidannotations.annotations.EActivity
import org.androidannotations.annotations.OptionsItem

/**
 * Wrapping activity for [AboutFragment].
 */
@SuppressLint("Registered")
@EActivity(R.layout.activity_about)
open class About : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    @OptionsItem(android.R.id.home)
    internal fun navUp() {
        NavUtils.navigateUpFromSameTask(this)
    }
}
